### ISSUE TITLE
[Doc] Contribution Guideline

### ISSUE THREAD

**Issue by Quorafind** [obsidian-community/web-clipper-templates#3]
Hi there! 

It would be great to include some contribution guidelines for the templates. This might cover things like naming conventions when adding templates to the repository, whether usage instructions should be included, and if any adjustment guidelines are necessary.

**Comment by sigrunixia** [comment]
This also has some crossover with #1 .

**Comment by esodesod** [comment]
Totally agree! I've added some comments in the discussion https://github.com/obsidian-community/web-clipper-templates/discussions/1#discussioncomment-10987234, mainly related to naming conventions.

Adding instructions to some of the templates is highly relevant as well, e.g. one will need to click on the "view transcript"-button on youtube videos to load the transcript, and for the clipper to actually be able to extract the transcript.

**Comment by sigrunixia** [comment]
> Adding instructions to some of the templates is highly relevant as well, e.g. one will need to click on the "view transcript"-button on youtube videos to load the transcript, and for the clipper to actually be able to extract the transcript.

Agreed. Seeing the comments in #4 , we and the end template user should know going in if this template is build around a certain plugin usage as well.

**Comment by sigrunixia** [comment]
I think I will start on the PR side first.

What we need:
- The template (.json)
- The template guide (.md)
- Ideally, a screenshot that shows the template post import. It doesn't need to be in default theme. (img, which we can make into a .webp).

This means we need a system that helps:
1. Keep track of the 3 needs
2. Reduces submission and processing repetition
3. Makes these easy to update

**Comment by DudeThatsErin** [comment]
Once this gets straightened out, I have more time on my hands now (got laid off again -.-) so I can write up some docs for the help site. :)

**Comment by aparente** [comment]
> Once this gets straightened out, I have more time on my hands now (got laid off again -.-) so I can write up some docs for the help site. :)

Did this ever get written?
